web/themes/joinup/templates/block/block--separated.html.twig

```diff
@@ -0,0 +1,45 @@
+{#
```

Comment:
Yes please, let's keep it simple with a function as long as only classes are managed.